packages/flutter_test/lib/src/binding.dart

```diff
@@ -194,24 +198,29 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   /// this method works when the test is run with `flutter run`.
   Future<Null> pump([ Duration duration, EnginePhase newPhase = EnginePhase.sendSemanticsUpdate ]);
 
-  /// Runs a [callback] that performs real asynchronous work.
+  /// Runs a `callback` that performs real asynchronous work.
```

Comment:
The advantage of putting them in brackets is that although dartdoc won't hyperlink them, IDEs see it as a reference to the argument, and refactoring tools will automatically rename these references.

Reply:
we need a way to distinguish them. The square brackets will suddenly start hyperlinking if something called "callback" is added to the scope, otherwise.